lib/ocpp/Kconfig

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2023 Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+config OCPP
```

Comment:
IMO this should be marked as Experimental, given it relies on our Websocket library (which is still considered as an experimental).